tests/CssToInlineStylesTest.php

```diff
@@ -60,7 +61,7 @@ public function testInlineStylesBlock()
         $this->cssToInlineStyles->setUseInlineStylesBlock();
         $this->cssToInlineStyles->setHTML($html);
         $actual = $this->findAndSaveNode($this->cssToInlineStyles->convert(), '//a');
-        $this->assertEquals($expected, $actual);
+        self::assertEquals($expected, $actual);
```

Comment:
using ``$this->assertEquals()`` is much more common in PHPUnit (and it is the documented way), even if the method is technically declared as static to be accessible both ways